PublicOnboardingDemo/projects/PublicOnboardingDemo.json

```diff
@@ -0,0 +1,225 @@
+{
+  "links" : [ ],
+  "name" : "PublicOnboardingDemo",
+  "options" : {
+    "description" : "Public Onboarding Demo\n\nThis contribution allows public users to sign up for a VANTIQ account without requiring a VANTIQ employee to manually send them a link. \n\nThis namespace contains two clients, the first is publicly accessibly and the second is private, and one procedure. The first client allows the user to input their name and email and register for a VANTIQ account. The client makes sure that the user has filled out all the fields and has a valid email address. Then, the user is sent an email with a link to the VANTIQ sign up form with keycloak. Once they have gone through the keycloak sign up process, they are redirected to the second client which is a landing page that tells the user what their account credentials are. \n\n",
```

Comment:
description should note that this only works in an Application Namespace, not a Developer namespace.